src/styles/star.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
```

Comment:
These styles are redundant if you use the `background` shorthand property. It's a good practice to use shorthand properties when you can, as it makes your code cleaner and easier to read. You can combine these three lines into one: `background: url(../images/star.svg) no-repeat center;`